cmr-exchange/ous-plugin/project.clj

```diff
@@ -15,28 +15,28 @@
        ns
        "\u001B[35m]\u001B[33m λ\u001B[m=> "))
 
-(defproject gov.nasa.earthdata/cmr-ous-plugin "0.3.0-SNAPSHOT"
+(defproject gov.nasa.earthdata/cmr-ous-plugin "0.3.1-SNAPSHOT"
   :description "A CMR services plugin that performs URL translations for subsetted GIS data"
   :url "https://github.com/cmr-exchange/cmr-ous-plugin"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0"}
   :dependencies [[cheshire "5.8.1"]
                  [clojusc/trifl "0.4.2"]
-                 [clojusc/twig "0.4.0"]
-                 [com.stuartsierra/component "0.3.2"]
+                 [clojusc/twig "0.4.1"]
+                 [com.stuartsierra/component "0.4.0"]
                  [environ "1.1.0"]
                  [gov.nasa.earthdata/cmr-authz "0.1.1"]
                  [gov.nasa.earthdata/cmr-exchange-common "0.2.2"]
                  [gov.nasa.earthdata/cmr-exchange-query "0.3.0-SNAPSHOT"]
                  [gov.nasa.earthdata/cmr-http-kit "0.2.0-SNAPSHOT"]
                  [gov.nasa.earthdata/cmr-jar-plugin "0.1.0"]
-                 [gov.nasa.earthdata/cmr-metadata-proxy "0.2.0-SNAPSHOT"]
+                 [gov.nasa.earthdata/cmr-metadata-proxy "0.2.2-SNAPSHOT"]
```

Comment:
Note: 0.2.1 was not used. some config params were removed from config.clj in 0.2.1.  This is why I modified the config.clj in a few places in this PR.